### ISSUE TITLE
typescript broken with plugin

### ISSUE THREAD

**Issue by daniluk4000** [nuxt/vite#33]
### Version
nuxt-vite: 0.0.8
nuxt: 2.15.2

### Configuration

```js
buildModules: [
    [
      'nuxt-vite',
      '@nuxt/typescript-build',
      {
        typeCheck: {
          typescript: {
            memoryLimit: 8000,
            extensions: {
              vue: true,
            },
          },
        },
      },
    ],
    'nuxt-typed-vuex',
  ],
```

### Description

The error happens in file where I import @nuxt/types like import { Plugin } from '@nuxt/types';

**Comment by pi0** [comment]
Hi @daniluk4000. You have to import from types like `import type { Plugin } from '@nuxt/types'`

**Comment by daniluk4000** [comment]
> 
> 
> Hi @daniluk4000. You have to import from types like `import type { Plugin } from '@nuxt/types'`

Thanks for quick answer! That didn't help :(

**Comment by pi0** [comment]
No worries but then i would need a reproduction repository to see what is wrong.

**Comment by daniluk4000** [comment]
@pi0 there you go: https://github.com/daniluk4000/nuxt-vite-bug

I have create empty app with nuxt-create-app, removed @nuxt/typescript-runtime and nuxt-ts

Well, error is now different: my file can't even be compiled

**Comment by pi0** [comment]
Thanks for reproduction. It seems typerscript (which should be done via vite>esbuild) is broken for plugins because of another workaround we use. Will investigate.

**Comment by pi0** [comment]
Issue fixed in ^0.0.10

**Comment by daniluk4000** [comment]
That was incredibly fast. Issue resolved in my local repo as well. Thanks!
